README.md

```diff
@@ -167,48 +178,110 @@ get_one_benchmark(
+
+Hereby, the mappings between shortened `benchmark_name` and actual benchmarks are:
+
+| `benchmark_name`   | Actual Benchmark                                   |
+|--------------------|----------------------------------------------------|
+| `"ae"`             | Amplitude Estimation (AE)                          |
+| `"dj"`             | Deutsch-Jozsa                                      |
+| `"grover-noancilla"` | Grover's (no ancilla)                              |
+| `"grover-v-chain"` | Grover's (v-chain)                                 |
+| `"ghz"`            | GHZ State                                          |
+| `"graphstate"`     | Graph State                                        |
+| `"portfolioqaoa"`  | Portfolio Optimization with QAOA                   |
+| `"portfoliovqe"`   | Portfolio Optimization with VQE                    |
+| `"qaoa"`           | Quantum Approximation Optimization Algorithm (QAOA) |
+| `"qft"`            | Quantum Fourier Transformation (QFT)               |
+| `"qftentangled"`   | QFT Entangled                                      |
+| `"qgan"`           | Quantum Generative Adversarial Network             |
+| `"qpeexact"`       | Quantum Phase Estimation (QPE) exact               |
+| `"qpeinexact"`     | Quantum Phase Estimation (QPE) inexact             |
+| `"qwalk-noancilla"` | Quantum Walk (no ancilla)                          |
+| `"qwalk-v-chain"`  | Quantum Walk (v-chain)                             |
+| `"realamprandom"`  | Real Amplitudes ansatz with Random Parameters      |
+| `"su2random"`      | Efficient SU2 ansatz with Random Parameters        |
+| `"twolocalrandom"` | Two Local ansatz with Random Parameters            |
+| `"vqe"`            | Variational Quantum Eigensolver (VQE)              |
+| `"wstate"`         | W-State                                            |
+| `"shor"`            | Shor's                                             |
+| `"hhl"`            | HHL                                                |
+| `"pricingcall"`          | Pricing Call Option                                |
+| `"pricingput"`          | Pricing Put Option                                 |
+| `"groundstate"`          | Ground State                                       |
+| `"routing"`          | Routing                                            |
+| `"tsp"`          | Travelling Salesman                                |
+
+
+
+For example, in order to obtain the *5*-qubit Deutsch-Josza benchmark on algorithm level, use the following:
 ```python
 from mqt.bench import get_one_benchmark
 
 qc = get_one_benchmark("dj", "alg", 5)
 ```
 
-In order to get access to all benchmarks, including the Qiskit application module algorithms, some addtional dependencies are required. 
-These can be installed via pip as follows:
+Additionally, this python package includes the same webserver used for the hosting of the 

```

Comment:
I'd make a separate section out of this. Something like "Locally hosting the MQT Bench Viewer"

The the description can be boiled down significantly. Something along the lines of

After installing the `mqt.bench` Python package via
```
(venv) $ pip install mqt.bench
```
the MQT Bench Viewer can be started from the terminal via
```
(venv) $ mqt.bench
```
This first searches for the most recent version of the benchmark files on GitHub and offers to download them.
Afterwards, the webserver is started locally.

Reply:
I agree, done.